### PR TITLE
add new recipe sbol-utilities 1.0a16

### PR DESCRIPTION
**Pull request by kenza12**
Add new recipe sbol-utilities 1.0a16 : https://github.com/SynBioDex/SBOL-utilities

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by guillaume-gricourt** [comment]
Please, don't merge this PR now

**Comment by breakthewall** [comment]
Hi, I'm one of the maintainers of this package. Thanks!

**Comment by tduigou** [comment]
Hello, I'm agree to be listed as mainainer. Thanks

**Comment by daler** [comment]
In general, we recommend [not including unit tests](https://bioconda.github.io/contributor/guidelines.html#tests) but primarily because they might cause timeouts. Doesn't seem to be the case here, and the rest of the recipe looks good. 

@guillaume-gricourt you mentioned not wanting to merge this yet, can you clarify what you want to finalize?

**Comment by guillaume-gricourt** [comment]
> you mentioned not wanting to merge this yet, can you clarify what you want to finalize?

Yes, a patch was applied properly 31bf2b4. The PR looks good to me.

**Comment by kenza12** [comment]
@BiocondaBot please add label

**Comment by bgruening** [comment]
@BiocondaBot please merge

**Comment by bgruening** [comment]
@kenza12 thanks!

**Comment by kenza12** [comment]
> I received an error uploading the build artifacts or merging the PR!

Thanks @bgruening ! Is the merge failing according to the bot ?

**Comment by daler** [comment]
@BiocondaBot please merge

File: recipes/sbol-utilities/patch [hunk 1]
```diff
@@ -0,0 +1,17 @@
+diff --git a/sbol_utilities/sbolgraph-standalone.js b/sbol_utilities/sbolgraph-standalone.js
```
Comment (bgruening) [inline recipes/sbol-utilities/patch:1]: this patch is not applied

**Comment by bgruening** [comment]
@kenza12 can you remove the patch, I don't think you are using it. At least its not specified in the meta.yaml and it should be there. Please trigger a rebuild. Then we can merge I think.

**Comment by kenza12** [comment]
> @kenza12 can you remove the patch, I don't think you are using it. At least its not specified in the meta.yaml and it should be there. Please trigger a rebuild. Then we can merge I think.

@bgruening , this patch was necessary to fix one error on bioconda. My colleague @guillaume-gricourt just added it to the recipe too. It's weird that it was working without the patch applied. I hope it will work now. Thank you.

**Comment by bgruening** [comment]
@BiocondaBot please fetch artifacts

**Comment by kenza12** [comment]
> Yes, very wired that it worked without it. Can you test the artefacts?

I downloaded the generated artifacts and tested this way : (I don't know if it's the good way but it seems working)
```
(test_artifacts) kenza@kenza-Precision-3551:~/Téléchargements$ mamba install -c packages LinuxArtifacts/packages/noarch/sbol-utilities-1.0a16-pyhdfd78af_0.tar.bz2

                  __    __    __    __
                 /  \  /  \  /  \  /  \
                /    \/    \/    \/    \
███████████████/  /██/  /██/  /██/  /████████████████████████
              /  / \   / \   / \   / \  \____
             /  /   \_/   \_/   \_/   \    o \__,
            / _/                       \_____/  `
            |/
        ███╗   ███╗ █████╗ ███╗   ███╗██████╗  █████╗
        ████╗ ████║██╔══██╗████╗ ████║██╔══██╗██╔══██╗
        ██╔████╔██║███████║██╔████╔██║██████╔╝███████║
        ██║╚██╔╝██║██╔══██║██║╚██╔╝██║██╔══██╗██╔══██║
        ██║ ╚═╝ ██║██║  ██║██║ ╚═╝ ██║██████╔╝██║  ██║
        ╚═╝     ╚═╝╚═╝  ╚═╝╚═╝     ╚═╝╚═════╝ ╚═╝  ╚═╝

        mamba (0.7.6) supported by @QuantStack

        GitHub:  https://github.com/mamba-org/mamba
        Twitter: https://twitter.com/QuantStack

█████████████████████████████████████████████████████████████

Preparing transaction: done
Verifying transaction: done
Executing transaction: done
(test_artifacts) kenza@kenza-Precision-3551:~/Téléchargements$ conda list
# packages in environment at /home/kenza/anaconda3/envs/test_artifacts:
#
# Name                    Version                   Build  Channel
_libgcc_mutex             0.1                 conda_forge    conda-forge
_openmp_mutex             4.5                       2_gnu    conda-forge
bzip2                     1.0.8                h7f98852_4    conda-forge
ca-certificates           2022.6.15            ha878542_0    conda-forge
ld_impl_linux-64          2.36.1               hea4e1c9_2    conda-forge
libffi                    3.4.2                h7f98852_5    conda-forge
libgcc-ng                 12.1.0              h8d9b700_16    conda-forge
libgomp                   12.1.0              h8d9b700_16    conda-forge
libnsl                    2.0.0                h7f98852_0    conda-forge
libuuid                   2.32.1            h7f98852_1000    conda-forge
libzlib                   1.2.12               h166bdaf_1    conda-forge
ncurses                   6.3                  h27087fc_1    conda-forge
openssl                   3.0.4                h166bdaf_2    conda-forge
pip                       22.1.2             pyhd8ed1ab_0    conda-forge
python                    3.10.5          ha86cf86_0_cpython    conda-forge
python_abi                3.10                    2_cp310    conda-forge
readline                  8.1.2                h0f457ee_0    conda-forge
sbol-utilities            1.0a16             pyhdfd78af_0    file:///home/kenza/T%C3%A9l%C3%A9chargements/LinuxArtifacts/packages
setuptools                62.6.0                   pypi_0    pypi
sqlite                    3.39.0               h4ff8645_0    conda-forge
tk                        8.6.12               h27826a3_0    conda-forge
tzdata                    2022a                h191b570_0    conda-forge
wheel                     0.37.1             pyhd8ed1ab_0    conda-forge
xz                        5.2.5                h516909a_1    conda-forge
zlib                      1.2.12               h166bdaf_1    conda-forge
(test_artifacts) kenza@kenza-Precision-3551:~/Bureau/test-sbol-utilities/SBOL-utilities$ python3
Python 3.10.5 | packaged by conda-forge | (main, Jun 14 2022, 07:06:46) [GCC 10.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import sbol_utilities

```

**Comment by bgruening** [comment]
@BiocondaBot please merge

**Pull request merged**